assets/css/main.css

```diff
@@ -136,6 +140,13 @@ textarea {
   margin-left: 19px;
 }
 
+.margin-right-19 {
+  margin-right: 19px;
+}
+
+.margin-right-5 {
```

Comment:
ditto

Reply:
Can i resolve this in my most resent PR?

Reply:
Sure